docs/getting-started/getting-started-with-cwl.rst

```diff
@@ -1,4 +1,4 @@
-.. note:: This tutorial assumes basic familarity with Docker as its example involves a Docker image. You may wish to start with :doc:`Getting Started With Docker </getting-started/getting-started-with-docker>` if you are not familiar with it.
+.. note:: This tutorial assumes basic familarity with Docker as its example involves a Docker image, and have access to a system that can run Docker. You may wish to start with :doc:`Getting Started With Docker </getting-started/getting-started-with-docker>` if you are not familiar with it.
```

Comment:
This sentence doesn't read quite right, there might be a grammar error.

Reply:
Looks like I used second-person implicit for the first part, then came back to the sentence and tacked on the second part without remembering the object and object-verb of the first part was implicit... hmmm

> This tutorial assumes [you have] basic familiarity... and have access..."

Curious if what I wrote is "grammatically correct but bad" vs "definitely incorrect," but either way, it's super confusing. Fixed it, thanks for catching this.